josemak25/hairx-client
# Ft regimen setup (home/hair goals screen)

#### What does this PR do?

Initial build and style of Home?Hair Goals selection page

#### Description of Task to be completed?

Have the home screen show the following:

1. A 'Hair Goals' header and explanation text.
2. An inline list of hair goal items.
3. A 'Start Regimen Setup' button

#### How should this be manually tested?

After cloning the repo,
`CD` into it and 'yarn install', then 'yarn start'


Using an emulator: Launch the app, and you should see the home screen after loading.

#### Screenshots

![Screenshot 2020-04-25 at 22 14 13](https://user-images.githubusercontent.com/8383707/80291141-dcc96980-8742-11ea-89bc-976f5014540f.png)
